code/datums/action.dm

```diff
@@ -708,19 +726,60 @@
 	button.maptext_height = 12
 	return button
 
+/datum/action/cooldown/Destroy()
+	if(!isnull(sequence_actions))
+		sequence_actions.Cut()
```

Comment:
`sequence_actions` contains no references and as such don't need to be handled in `Destroy()` as far as I can tell?